### DISCUSSION TITLE
Please add a way to configure some system prompts

### DISCUSSION THREAD

**Discussion by jamesliu4c** [community/community#122994] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Please add a way to add system prompts for Copilot. I would like to always say this, but I don't want to type it:
- I don't need to be reminded that you are an AI chatbot. I understand that I need to review any code snippets before using them in production.
- Be concise
- Use bash best practices, such as using shellcheck

**Comment by pm7y** [comment]
I think this would be a great addition. Something like an `.editorconfig` file, which is a well-known file the IDE knows to look for. Something similar would be really useful to have some baseline prompts and settings for Copilot that could be used across a team.
